### PR TITLE
mpd + libnfs

### PR DESCRIPTION
**Pull request by bismark**
At the risk of revisting #3337...

Adds libnfs as a new formula and updates mpd to allow it as an optional dependency.

### CODE REVIEW

File: Library/Formula/libnfs.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  depends_on "autoconf" => :build
+
+  def install
+    system "./bootstrap"
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+
+    system "make", "install"
+  end
```
Comment (bfontaine) [inline Library/Formula/libnfs.rb]: Any chance you could add a test? Thanks!

Comment (bismark) [inline Library/Formula/libnfs.rb]: added!

**Comment by bfontaine** [comment]
Can someone from @Homebrew/owners give their :+1: here?

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**